test/bench/granular/date.bench.ts

```diff
@@ -17,15 +18,17 @@ async function main() {
   const suite = new Suite('Date');
   const testDocs = await getTestDocs('date');
 
-  for (const operation of OPERATIONS) {
-    for (const documentPath of testDocs) {
+  for (const documentPath of testDocs) {
+    const tags = getTypeTestTags(documentPath);
+    for (const operation of OPERATIONS) {
```

Comment:
Not immediately obvious what flipping the order of the loops does?

Reply:
I just didn't want to call `getTypeTestTags` more often than we needed to since the only parameter is the document path. It's admittedly a micro-optimization since this is just the benchmark run setup